jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/CreateLambdaFunction.kt

```diff
@@ -9,16 +9,21 @@ import com.intellij.openapi.actionSystem.LangDataKeys
 import icons.AwsIcons
 import software.amazon.awssdk.services.lambda.model.Runtime
 import software.aws.toolkits.jetbrains.services.iam.IamRole
-import software.aws.toolkits.jetbrains.services.lambda.orThrow
 import software.aws.toolkits.jetbrains.services.lambda.runtime
 import software.aws.toolkits.resources.message
 
-class UploadLambdaFunction(private val handlerName: String) : AnAction(message("lambda.create_new"), null, AwsIcons.Actions.LAMBDA_FUNCTION_NEW) {
+class CreateLambdaFunction(private val handlerName: String? = null) : AnAction(message("lambda.create_new"), null, AwsIcons.Actions.LAMBDA_FUNCTION_NEW) {
     override fun actionPerformed(event: AnActionEvent) {
         val project = event.getRequiredData(LangDataKeys.PROJECT)
-        val runtime = event.runtime().orThrow
+        val runtime = event.runtime()
```

Comment:
Can delete the extension function then - that was the only place it was used I believe.

Reply:
Oh, didnt even look